src/_layouts/default.html

```diff
@@ -6,7 +6,7 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-
+    <link rel="shortcut icon" type="image/x-icon" href="/assets/favicon.ico?">
```

Comment:
Is the `?` in the `href` meant to be there?

Reply:
No, good catch!